### ISSUE TITLE
Create blockstack forum post pointing to github repos

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/resources#26]
The information in Github is well organized but a forum user looking at the working groups / governance category would have no idea how to find it. We should create a canonical post listing out the repositories, their purpose, and some evergreen links to direct people from the forum to the repo.

**Comment by lrettig** [comment]
Makes sense. It can be pinned to the top of the category on Discourse. Jenny should be able to help us with this.

**Comment by whoabuddy** [comment]
I created a comment on the [About the Governance Category](https://forum.blockstack.org/t/about-the-governance-category/10376) forum post, which is pinned in the Working Group > Governance category.

@blockstackers Do you think we should do anything else to link the different outlets (Github/Discourse/Discord), or can we close this issue?
